index.html

```diff
@@ -728,7 +776,7 @@
 
 			function fillpopup_rfp(data){
 				var html_rfp = "";
-				html_rfp = html_rfp + "<span class='varname'>School: </span> <span class='attribute'>" + data.account_name + "</span>";
+				html_rfp = html_rfp + "<span class='varname'>School: </span> <span class='attribute'>" + standardizeCase(data.account_name) + "</span>";
```

Comment:
Currently only three fields are getting standardized (this one and two in the RSL equivalent below), because where we have clean text I'd rather not risk breaking anything by changing it.  If you want to add this normalisation to any other strings, just wrap them in the function call like this

Reply:
That makes sense, and I will let RYHT know that your standardization solution can be extended to additional spreadsheet sources